app/src/main/java/fi/thl/koronahaavi/guide/NotificationGuideViewModel.kt

```diff
@@ -0,0 +1,41 @@
+package fi.thl.koronahaavi.guide
+
+import androidx.annotation.DrawableRes
+import androidx.annotation.StringRes
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import androidx.lifecycle.map
+import fi.thl.koronahaavi.R
+
+class NotificationGuideViewModel : ViewModel() {
+    private val currentPage = MutableLiveData(0)
+    fun currentPage(): LiveData<Int> = currentPage
+
+    private fun currentPageValue(): Int = currentPage.value ?: 0
+
+    fun updatePage(index: Int) {
+        if (index != currentPage.value && index in pages.indices) {
+            currentPage.postValue(index)
```

Comment:
Voice over: would it be possible to move the focus to next/prev page's text view? Now it jumps to the "previous" button.

Reply:
Maybe.. but I'm finding some docs about avoiding manually changing focus, in order to keep navigation consistent. Good point about checking accessibility though, I did add content description to the page number icon, since could not figure out how to make ViewPager2 automatically announce current page.. not sure whats missing there, focusable or importantForAccessibility or contentDescription did not help